src/file-picker/test/index.js

```diff
@@ -109,24 +107,10 @@ test('handles 2 files selected', t => {
     }
   }
   component.find(`.${CLASS_PREFIX}-file-input`).simulate('change', e)
-  t.deepEqual(component.state('files'), e.target.files)
   t.is(component.find(`.${CLASS_PREFIX}-text-input`).prop('value'), '2 files')
   t.true(component.find(`.${CLASS_PREFIX}-button`).contains('Replace files'))
 })
 
-// Firefox returns the same array instance in each change event for some reason
-test('clones files array', t => {
```

Comment:
Happy to mock out `useState` implementation, but unsure of the true value of the test.